### ISSUE TITLE
Db migration issue: No DbContext was found in assembly

### ISSUE THREAD

**Issue by relliky2d** [simplcommerce/SimplCommerce#692]
Microsoft Visual Studio Community 2017 (v15.9.5)
.NET Core v2.2.103

Solution builds successfully, but migration fails:

PM> Update-Database
No DbContext was found in assembly 'SimplCommerce.Infrastructure'. Ensure that you're using the correct assembly and that the type is neither abstract nor generic.

**Comment by thiennn** [comment]
In the Package Manager Console, The Default Project should be src/SimplCommerce.WebHost

**Comment by relliky2d** [comment]
Yeah I saw that, and changed it -- but then I get the following on 'Update-Database'. Still builds successfully.

"Could not load assembly 'SimplCommerce.WebHost'. Ensure it is referenced by the startup project 'SimplCommerce.Infrastructure'."
